utils/error-utils.js

```diff
@@ -1,4 +1,4 @@
-const DEBUG = false;
+const DEBUG = true;
```

Comment:
if this is merged to master, will this need to be set as true?

Reply:
fixed